packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -19,7 +19,8 @@ import { Popper } from '../../helpers';
 
 export interface SearchAttribute {
   /** The search attribute's value to be provided in the search input's query string.
-   * It should have no spaces and be unique for every attribute.
+   * It should be unique for every attribute. If spaces are used they will get wrapped
+   * in single quotes in the search input string.
```

Comment:
I don't think this is the correct prop description to update. This is referring to the attribute names which should be unique with no spaces. Like 'username', not the values associated with usernames.

Reply:
The attribute names are just bellow, the 'displayName' property. This is the value of the attributes in the form field.

So I document that if someone uses spaces in the attribute values, these will get wrapped in quotes within the search string.

Open to change it if you have some suggestion!

Reply:
actually, the attribute names and the display names are all part of the same concept related to attributes, not the attribute values. 

This way, when someone wants a field on the advanced search form to be displayed as `User name` (with a space) but then the corresponding attribute displayed in the input before the value is using the attr prop to display `username`. so neither the SearchAttribute's `attr` field, nor its `display` field have anything to do with the associated values.

Reply:
I don't think we have a place where the docs state any rules about the values themselves having no spaces